PantryPal/StorageType.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {View,Text} from 'react-native';
+import {SelectList, MultipleSelectList }from 'react-native-dropdown-select-list'
+
+const App = () => {
```

Comment:
We already have a function called App in PantryPal/App.tsx, and it contains the navigation stack for our app. Please change this to a different name, preferably it should match the name of the file. In this case, StorageType. See other screens for reference.